Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -78,13 +74,21 @@ inline CGPoint __CreateCGPointWithTransform(CGFloat x, CGFloat y, const CGAffine
         return _impl.startingPoint;
     }
 
+    void SetIsFigureClosed(bool closed) {
```

Comment:
nit: would make more sense to have a CloseFigure which closes it and sets isFigureClosed